src/Console/Command/SelfUpdateCommand.php

```diff
@@ -43,7 +43,8 @@ protected function configure()
                 )
             )
             ->setDescription('Update php-cs-fixer.phar to the latest stable version.')
-            ->setHelp(<<<'EOT'
+            ->setHelp(
+                <<<'EOT'
```

Comment:
Heredoc arguments can't be anything else than multi-line arguments, because as you can't write this: https://3v4l.org/RUlJB